po/ko.po

```diff
@@ -750,24 +763,27 @@ msgstr "테스트 모듈"
 #: src/SUMMARY.md:226
 #, fuzzy
 msgid "Rust Bridge"
-msgstr "러스트 바이너리"
+msgstr "C++ 브리지"
```

Comment:
This is severely wrong. Even if you don't understand Korean, you should know that this is a wrong translation. There's no C++ in the original text.